pkg/test/resourcefixture/testdata/basic/workstations/workstationcluster/workstationcluster-full/update.yaml

```diff
@@ -0,0 +1,39 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: workstations.cnrm.cloud.google.com/v1alpha1
+kind: WorkstationCluster
+metadata:
+  name: workstationcluster-${uniqueId}
+spec:
+  projectRef:
+    external: ${projectId}
+  location: us-west1
+  displayName: workstationcluster-${uniqueId}-displayname
+  annotations:
+    - key: a-key1
+      value: a-value1
+    - key: a-key2
+      value: a-value2
+  labels:
+    - key: l-key1
+      value: l-value1
+    - key: l-key2
+      value: l-value2
```

Comment:
Not sure if that is desirable actually..? This way we don't have to filter out any kcc-specific labels or annotations. It's purely just what the user specifies.

Reply:
Here's how labels and annotations are suggested to be used: to [apply to a group of resources that have logical relationships](https://cloud.google.com/compute/docs/labeling-resources).   
For KCC users, that is the metadata labels and annotations.  The way k8s users configure the labels and annotations are not via manually adding the metadata.annotation, but using tools like kustomize or kubectl to apply a bulk change.

Reply:
Hmm, would we potentially want to revisit this design decision for new direct resources? I wonder if we really want to map k8s labels / annotations directly to gcp labels / annotations. One of the issues a customer recently hit for SQLinstance direct controller was because we were applying the labels from the KRM resource directly to GCP (https://github.com/GoogleCloudPlatform/k8s-config-connector/issues/2706#issuecomment-2357595555)

Reply:
yeah, I think computeforwardingrule has the same issue.  Fortunately the set-labels is a separate GCP call, so we caught it in an earlier stage. @gemmahou  

Maybe modifying the resource in the base controller level.